### ISSUE TITLE
Expose canonicalization endpoints

### ISSUE THREAD

**Issue by bobmorley** [Apicurio/apicurio-registry#2473]
We have a use case where it would be useful to be able to compare a version of a JSON schema that our producer uses, to the version that has been registered in Apicurio. While there are a couple of endpoints that indicate it uses a canonicalized/protocol specific version of the schema, there is no endpoint that allows me to `GET` the canonicalized schema from an arbitrarily provided schema.

Can this be considered from an enhancement?

**Comment by EricWittmann** [comment]
Yes it can!  Sorry for the delay... my backlog getting out of hand again.

This is an interesting feature request.  I don't see any reason why we can't introduce a new endpoint or perhaps a query param on an existing endpoint.

**Comment by carlesarnal** [comment]
@EricWittmann I think we should consider this for Registry V3.
